src/shims/foreign_items.rs

```diff
@@ -527,7 +538,14 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
                 this.write_scalar(Scalar::from_u32(n.to_bits()), dest)?;
             }
 
-            "cbrt" | "cosh" | "sinh" | "tan" | "acos" | "asin" | "atan" => {
+            | "cbrt"
+            | "cosh"
+            | "sinh"
+            | "tan"
+            | "acos"
+            | "asin"
+            | "atan"
+            => {
```

Comment:
opinions on this style? I like it because it makes changing the list never cause fallout in unrelated elements

Reply:
Not all lines are consistent though, the first line is different. So the thing that would be most like how we format comma-separated lists would be to make every line `"foo" |`. Unfortunately though, [this does not compile](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=91f5a9e520b3e4ef5af878f66da38177).

So the version taht compiles would be to make every line `| "foo"`, I guess?

Reply:
Looks like that's what you got. So in the confines of the Rust grammar, I guess that is the best we'll get.

Shame that rustfmt destroys that. :/